reports/wip-limits.ts

```diff
@@ -15,14 +15,13 @@ export function getDefaultConfiguration(): any {
 }
 
 interface WIPLimitsLineItem {
-    name: string,
     count: number,
     limit: number
 }
 
 interface WIPLimitsReport {
     name: string,
-    lineItems: WIPLimitsLineItem[]
+    stages: { [key: string]: WIPLimitsLineItem }
```

Comment:
`if (actual <limit)` logic in should be in process.  So that means this is `{[key: string]: string}` and process would set the value to `OK`, `Warning` or `Critical`.  Then render just switches on string to emoji (rendering).  Biz logic is in process.

That allows us to introduce htmlRendering and the biz logic won't be in two places.  It will just be a matter of how markdown renders 'OK' and how html chooses to render 'OK'